### ISSUE TITLE
[Request]: New method card: Defining Service Measures & Managing Data

### ISSUE THREAD

**Issue by amandarkennedy** [18F/guides#490]
### A description of the work

**Note:** This ticket originated in the service design sub-chapter's Trello board. It will likely need further refinement before it is prioritized.

### Point of contact on this issue

@katefisher808 (original author)

### Skills Needed

- [ ] Any Human
- [X] Design
- [ ] Content
- [ ] Engineering
- [ ] Acquisition
- [ ] Product
- [ ] Other

### Does this need to happen in the next 2 weeks?

- [ ] Yes
- [X] No

### How much time do you anticipate this work taking?

2-3 days

### Task List

- [x] Draft a method card for Defining Service Measures [using this template](https://docs.google.com/document/d/1gBNvcB-Wtn0xqiM1-yEhhfcdE6ghcI6UXjWSGg7l3Do/edit#heading=h.aou5xt3rvfpf)
- [x]  Determine which Methods section it should live under. 
- [x]  Get the content and IA reviewed. This could be either reviewed by one of the service design supervisors, @elisaachen @MelissaBraxton @lauranash18f, or someone from product.

### Acceptance Criteria
- [ ]  Publish the new method card.

**Comment by amandarkennedy** [comment]
Hi @katefisher808 — I transferred this over from the service design trello board. Not urgent — feel free to add more context to the ticket description!

**Comment by bpdesigns** [comment]
Hi @katefisher808 is this still current? There is another ticket for a [user story method issue](https://github.com/18F/guides/issues/489).

**Comment by elisaachen** [comment]
Hi @bpdesigns ! Not to speak for Kate by any means, but I think the user story method is different than this ticket. I think in Amanda's acceptance criteria, the reference of "user story" was more an action item to articulate a user story for defining a service measure. I know @katefisher808 is OOO this week so might expect a delayed response.

**Comment by amandarkennedy** [comment]
Thanks @elisaachen! @bpdesigns it is also possible that I may have copy/pasted the acceptance criteria from the user story ticket. So we should keep both tickets. Thanks!

**Comment by bpdesigns** [comment]
Thanks @amandarkennedy @elisaachen!

**Comment by MelissaBraxton** [comment]
This is an interesting idea! I could certainly see adding **service metrics** as a "validate" method, but I'd like to have a better sense of the content. (And I'm intentionally using the word metric here. "Measures" implies the actual number, whereas "metric" is about establishing what you'll measure and how you'll assess progress, which is I think what we want here.)

I'd think it could be pretty tough to cover both identifying metrics and also managing data in a single card, and I'd also suspect that data management would be hard to write about in a way that is both concise enough for a method card and also detailed enough to be actionable? 

Consider whether more there's a good justification for service metrics and data management as a section in a guide? 

Last caveat, I believe my role as content owner is to review/approve content, but I believe prioritization decisions / whether this is the right work to take on right now belong to @adunkman.

**Comment by katefisher808** [comment]
Thanks all! Took me a minute to re-remember what this trello card was about from a past service design brainstorm awhile back. Yes Elisa and Melissa are spot-on, the initial idea was more about defining metrics (and metrics of success) across a service experience. and probably less so on the managing data side of things. I'll update the description of this ticket with some additional thinking.

**Comment by bpdesigns** [comment]
Thank you @katefisher808 !!

**Comment by katefisher808** [comment]
I don't believe I have the authority to edit the initial post, but sharing an overview + three general sources of inspiration for the method itself. Happy to be a contributor/supporter on this if it is prioritized within TLC.

Overview: To learn how well our services are working, we need to measure short, medium, and long-term outcomes across the different moments in a service experience.

Method: Outlining short, medium, and long-term outcomes and corresponding metrics that help to measure progress toward those outcomes

Inspiration: 
- https://hodigital.blog.gov.uk/2017/08/04/what-we-mean-by-service-outcomes-and-measurement/
- https://design-kit-staging.s3.amazonaws.com/procedures/worksheets/000/000/068/original/DesignKit_indicatormadlibs_worksheet.pdf?1590455790
- https://www.gov.uk/service-manual/measuring-success/how-to-set-performance-metrics-for-your-service
- https://civicservicedesign.com/focus-on-impact-setting-measures-of-success-76dd207dbaf0

**Comment by bpdesigns** [comment]
@elisaachen if you have time to review - I'd like one SD supervisor to review this content

**Comment by elisaachen** [comment]
@bpdesigns  - I can take a review 6/6 or the following Monday!

**Comment by elisaachen** [comment]
@katefisher808  - If you can use scrubbed deliverables from your work on IRS DF as examples here, that would be fantastic. To Melissa's earlier point, I think adding in corresponding content in our UX Guide can be helpful too and worth a review of our guide to see if we can't create a TLC ticket out of that. @bpdesigns  - LGTM and feel like Kate can run with this when she joins TLC.

**Comment by katefisher808** [comment]
Sounds great! I'll assign myself, move this ticket status to 'increment goal' in TLC. Does that sound right @bpdesigns ?

**Comment by bpdesigns** [comment]
@katefisher808 that sounds good. Consider reaching out to @mitogi to pair or shadow this service design task.

**Comment by katefisher808** [comment]
Sounds good! @mitogi want to pair on this one? I'll most likely tackle two other TLC tasks first and start this one next week.

**Comment by mitogi** [comment]
Sure, @katefisher808! Looking forward to working together. 😄

**Comment by katefisher808** [comment]
Linking the placeholder doc for the method: https://docs.google.com/document/d/1Kjg5nYHXHRR6qqfMJbp3Q562gwrxLIcYd3PvQEU51aw/edit

**Comment by mitogi** [comment]
@elisaachen @MelissaBraxton @lauranash18f
 
 Hi, could one of you review this method card draft?
[[https://docs.google.com/document/d/1Kjg5nYHXHRR6qqfMJbp3Q562gwrxLIcYd3PvQEU51aw/edit]](https://docs.google.com/document/d/1Kjg5nYHXHRR6qqfMJbp3Q562gwrxLIcYd3PvQEU51aw/edit)

**Comment by bpdesigns** [comment]
@mitogi I reviewed the draft and the content is good. Do you feel comfortable implementing this card or would you like to pair with someone to make it live?

**Comment by mitogi** [comment]
I bet I can implement it. :D :D

**Comment by mitogi** [comment]
Created a pull request and waiting on @MelissaBraxton approval.
https://github.com/18F/guides/pull/630

**Comment by MelissaBraxton** [comment]
The content is too long to fit the printable card template. We'll need to make it more concise. 

![Image](https://github.com/18F/guides/assets/20651166/6b247e21-419d-4a44-bd16-2c1057104b0d)

**Comment by MelissaBraxton** [comment]
See my comment above. Not sure why this was closed? This needs to be corrected.

**Comment by mitogi** [comment]
I'll open

**Comment by bpdesigns** [comment]
@mitogi added one comment in your PR to see if we can fit a little more content back in. No worries if not!

**Comment by mitogi** [comment]
@bpdesigns added the content back in:
https://github.com/18F/guides/pull/648
